gen/models/resource-properties/sns-subscription.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "SNSTopicSubscription",
```

Comment:
Had to name it "SNSTopicSubscription" because "SNSSubscription" name is already taken by the "AWS::SNS::Subscription" resource. Just checking to see if i'm breaking any current naming conventions.